control-plane/csi-driver/src/bin/controller/controller.rs

```diff
@@ -133,6 +138,25 @@ async fn issue_fs_unfreeze(endpoint: String, volume_id: String) -> Result<(), St
     }
 }
 
+#[tracing::instrument]
```

Comment:
Though do we even need it?

```suggestion
#[tracing::instrument(err, skip_all)]
```